boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832.dts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2017 Shawn Nock <shawn@monadnock.ca>
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2021 morsisko
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
```

Comment:
```suggestion
```

Reply:
When this line is removed, project can't be compiled. Following error is displayed
` -- Found BOARD.dts: /__w/zephyr/zephyr/boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832.dts
devicetree error: /__w/zephyr/zephyr/dts/common/skeleton.dtsi:9 (column 1): parse error: expected '/dts-v1/;' at start of file`

Reply:
It seems correct to me to keep this line here in the file; why did you suggest this @gmarull?